Classes/Exporter/AbstractExporter.php

```diff
@@ -88,20 +71,23 @@ abstract class AbstractExporter
 	 */
 	protected $objectManager = null;
 
+    /**
+     * @var RenderingContext
+     */
+    protected $renderingContext = null;
+
 	/**
 	 * @param ObjectManager $objectManager
 	 */
-	public function __construct(ObjectManager $objectManager)
+	public function __construct(ObjectManager $objectManager, RenderingContext $renderingContext)
```

Comment:
Is the `RenderingContext` ever used? If it does not serve any purpose, it would be great if you could remove it.

Reply:
> @mhirdes Thanks again for your contribution! It seems however not to be working as soon as the date column is added to the export. Do you have time to look into this and update the PR?

Currently not.